setup.py

```diff
@@ -93,7 +94,9 @@
             "Programming Language :: Python :: 3",
         ],
         entry_points={
-            "console_scripts": ["charmcraft = charmcraft.main:main"],
+            "console_scripts": [
+                "charmcraft = charmcraft.main:main",
+            ],
```

Comment:
This change is not really a change, right?

Reply:
Nope, just leftover formatting delta from prior change. Will fix

Reply:
Fixed